TLM/TLM/UI/TrafficManagerMode.cs

```diff
@@ -0,0 +1,6 @@
+namespace TrafficManager.UI {
+    public enum TrafficManagerMode {
```

Comment:
Should this be moved in to State folder?

Reply:
Currently this enum is used only in UIBase class so UI folder is not so bad

Reply:
@kvakvs This is only remaining issue on this PR I think

Reply:
It has been moved close to where it is used. Is this an issue?

Reply:
Ah, sorry, didn't realise. All good!